lib/web/mage/translate-inline-vde.css

```diff
@@ -1,204 +0,0 @@
-/**
- * Copyright © Magento, Inc. All rights reserved.
- * See COPYING.txt for license details.
- */
-
-.trnslate-inline-area [data-translate],
```

Comment:
Looks like it's still used in 
https://github.com/magento/magento2/blob/9756182a77a472be52ce995cb080ea742d3cd0ef/app/code/Magento/Translation/view/adminhtml/templates/translate_inline.phtml#L79
https://github.com/magento/magento2/blob/9756182a77a472be52ce995cb080ea742d3cd0ef/app/code/Magento/Translation/view/frontend/templates/translate_inline.phtml#L68

Reply:
I've already checked, this file is used instead `magento/lib/web/mage/translate-inline.css`.